test/index.js

```diff
@@ -190,3 +192,27 @@ test("works with no styles at all", function(t) {
 
   t.end()
 })
+
+test("works with caching", function(t) {
+  var opts = {path: importsDir, cacheDir: cacheDir}
+  var cssFileName = path.resolve("test/fixtures/relative-to-source.css")
+  var postcssOpts = {from: cssFileName}
+  var css = fs.readFileSync(cssFileName)
+  var output = postcss()
+    .use(atImport(opts))
+    .process(css, postcssOpts)
+    .css.trim()
+
+  var imports = JSON.parse(fs.readFileSync(cacheDir + "/imports.json"))
+  var cacheFileName = imports[cssFileName].cache;
+  var cache = fs.readFileSync(cacheFileName, "utf8").trim()
+
+  t.equal(output, cache, "should put output in cache")
```

Comment:
we know here that the cache is written, how do we know the cache is being read after that ?